Upstream-Status: Inappropriate [Configuration]

Don't show "sfs_init(): Warning: file /etc/mc/sfs.ini not found",
because it's packaged separately.

Signed-off-by: Andreas Oberritter <obi@opendreambox.org>

--- mc-4.8.1/src/vfs/sfs/sfs.c.orig	2012-05-10 15:48:05.405830465 +0200
+++ mc-4.8.1/src/vfs/sfs/sfs.c	2012-05-10 15:48:24.389830919 +0200
@@ -419,7 +419,6 @@
 
     if (cfg == NULL)
     {
-        fprintf (stderr, _("%s: Warning: file %s not found\n"), "sfs_init()", mc_sfsini);
         g_free (mc_sfsini);
         return 0;
     }
